### PR TITLE
Navigation configuration

### LINKED ISSUE #2: Screen navigation configuration
## DESCRIPTION

> The objective of this issue is to configure the Stack Navigator in the project to be able to navigate between screens.

```[tasklist]
### Tasks
- [x] Install React Navigation.
- [x] Install required dependencies.
- [x] Make the necessary configurations.
- [x] Create new screens.
- [x] Configure bottom navigation.
```

### PR DESCRIPTION
**Pull request by landillo**
# Navigation configuration

We made the corresponding navigation setup for the application, installed the necessary dependencies up to this point, and created empty 'welcome' and 'login' screens to test navigation between them. We also configured project's structure.

### CODE REVIEW

**Review by montsegv-2 — APPROVED** [review]
LGTM :D

**Review by AlbertoG22 — APPROVED** [review]
Excellent work, team!

File: src/components/ButtonPrimary/styles.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from "react-native"
+
```
Comment (wiltrouble) [inline src/components/ButtonPrimary/styles.ts:2]: remove this row

File: src/navigation/StackNavigator.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import WelcomeScreen from '../screens/WelcomeScreen/WelcomeScreen';
+import LoginScreen from '../screens/LoginScreen/LoginScreen';
+
```
Comment (wiltrouble) [inline src/navigation/StackNavigator.tsx:5]: Remove this row

File: src/navigation/StackNavigator.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+const Stack = createStackNavigator();
+
+export const StackNavigator = () => {
+  return (
+    <Stack.Navigator>
+      <Stack.Screen name="Welcome" component={ WelcomeScreen } />
+      <Stack.Screen name="Login" component={ LoginScreen } />
+
```
Comment (wiltrouble) [inline src/navigation/StackNavigator.tsx]: remove this row

File: src/screens/WelcomeScreen/WelcomeScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <SafeAreaView>
+      <Text>Hola crayola</Text>
+      <ButtonPrimary 
+      title='Get started'
+      onPress={() => navigation.navigate('Login')}/>
+    </SafeAreaView>
+  );
+};
+
```
Comment (wiltrouble) [inline src/screens/WelcomeScreen/WelcomeScreen.tsx:18]: remove this row

**Review by wiltrouble — CHANGES_REQUESTED** [review]

**Review by wiltrouble — APPROVED** [review]

**Pull request merged**